packages/eslint-config-typescript/eslint-config-typescript.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+  plugins: ['@typescript-eslint'],
```

Comment:
Do we need to add settings for `import/extensions` to override including TS extensions?

```js
    'import/extensions': [
      'error',
      'ignorePackages',
      {
        js: 'never',
        mjs: 'never',
        jsx: 'never',
        ts: 'never',
        tsx: 'never',
      },
    ],
```

Reply:
I looked at the [docs for `import/extensions`](https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/extensions.md) and I'm not positive that we need to add that override since we do not use extensions in our file paths. It doesn't look like we are doing that in `@side/eslint-config-react` or in `audit-ui`, but perhaps we should. What do you think?

When I tested this config on audit-ui running locally, it didn't have trouble picking up those files.

Reply:
We are doing it in the services though – https://github.com/reside-eng/audit-service/blob/main/.eslintrc.js#L11-L18. This might not be needed in the Next.js side because I think Next's ESLint config is doing that under the hood.

Reply:
@hershmire would it help if I did some analysis on these settings; i.e. removing them from audit-service locally and seeing if everything still works?

Reply:
Sure, but I think they will be needed because our backend services are not getting that set anywhere as opposed to the UIs that are using Next ESLint configuration.

Reply:
@hershmire you are correct, we do need to set those settings in order for this config to work in non-Next.js repos. I will update this PR with the necessary settings.